src/reconciliation/manifest.ts

```diff
@@ -0,0 +1,24 @@
+import {Catalog, IRI} from '@netwerk-digitaal-erfgoed/network-of-terms-catalog';
+
+export class ServiceManifest {
+  public readonly versions = ['0.1', '0.2'];
+  public readonly schemaSpace = 'http://www.w3.org/2004/02/skos/core#Concept';
```

Comment:
Is this correct?

Reply:
Interesting :) The spec appears to only allow one URI. But what if a recon service uses more than one? The NoT mostly uses SKOS, but we also have a bit of RDFS (and perhaps we need another ontology in the future). We could point to a general page with info about our schema? ('This URI also MAY resolve to a page describing these properties and their identifiers.')

Reply:
Based on the specs, we decided to keep this for now:

> A URI which identifies the type of an [entity](https://reconciliation-api.github.io/specs/latest/#dfn-entity) returned by the reconciliation service, for example http://www.w3.org/2004/02/skos/core#Concept or https://schema.org/Thing. This URI MAY resolve to a page describing this type.